docs/troubleshooting.asciidoc

```diff
@@ -2,8 +2,8 @@
 == Troubleshooting
 
 Is something not working as expected?
-Don't worry if you can't figure out what the problem is; we’re here to help!
-As a first step, ensure your app is compatible with the agent's <<supported-tech,supported technologies>>.
+Don't worry if you can't figure out what the problem is, we’re here to help!
```

Comment:
I would keep the semicolon.

Reply:
Incidentally here's a good topic about comma splices: https://www.grammarly.com/blog/comma-splice/